src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -510,6 +526,8 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(
   // has been started.
   auto valid_startup_hook_callsite = true;
   if (module_metadata->assemblyName == "System"_W ||
+      module_metadata->assemblyName == "System.Net.Http"_W ||
+      module_metadata->assemblyName == "Microsoft.WindowsAzure.WebSites.Diagnostics"_W ||
```

Comment:
Non blocking, but this seems like something that will end up having many more entries in the future, I wonder if we can head that off somehow.

Reply:
Yeah, I understand your concern. I'm hoping that this list doesn't grow but if it does we can try to organize a solution for it then outside of this PR